File: 백준/Bronze/15552. 빠른 A＋B/빠른 A＋B.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder sb = new StringBuilder();
        int t = Integer.parseInt(br.readLine());
        for(int i = 0; i < t; i++) {
            String[] arr = br.readLine().split(" ");
            sb.append(Integer.parseInt(arr[0]) + Integer.parseInt(arr[1]) + "\n");
        }
        System.out.print(sb.toString());
    }
}

File: 백준/Bronze/15552. 빠른 A＋B/빠른 A＋B.js
var fs = require('fs');
var input = fs.readFileSync('/dev/stdin').toString().trim().split('\n');
var t = parseInt(input[0]);
var ans = '';
for (var i = 0; i < t; i++) {
  ans +=
    parseInt(input[i + 1].split(' ')[0]) +
    parseInt(input[i + 1].split(' ')[1]) +
    (i != t - 1 ? '\n' : '');
}
console.log(ans);